datavec/datavec-python/src/main/java/org/datavec/python/PythonJob.java

```diff
@@ -50,7 +61,7 @@ public void setup() throws Exception {
             }
             runF = PythonExecutioner.getVariable("run");
             if (runF.isNone() || !Python.callable(runF)) {
-                throw new RuntimeException("run() method not found!");
+                throw new Exception("run() method not found!");
```

Comment:
This still isn't right. I want you to create a custom python exception we can catch. You should be extending Exception to do that.